modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -290,365 +345,498 @@ Removes a reference to an entity.
 
 Call Action:
 
-* Use Web API Actions - https://msdn.microsoft.com/en-us/library/mt607600.aspx
+* Use Web API Actions - `+https://msdn.microsoft.com/en-us/library/mt607600.aspx+`
+
 
+==== Parameters
 
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
-| Action Name a| String |  The action name that is going to be called |  | x
-| Bounded Entity Type a| String |  An optional bounded entity type. It must be provided if the action is bounded. |  |
-| Bounded Entity Id a| String |  An optional bounded entity ID. It must be provided if the action is bounded. |  |
-| Related Entities a| Object |  This is the body that will be send in the request as a Map. |  `#[payload]` |
-| Target Variable a| String |  The name of a variable on which the operation's output will be placed |  |
-| Target Value a| String |  An expression that will be evaluated against the operation's output and the outcome of that expression will be stored in the target variable |  `#[payload]` |
+| Action Name a| String |  +++The action name that is going to be called+++ |  | x
```

Comment:
maybe this is just like the 'wills', but 'that is going to be called' could be just 'to be called'.